### PR TITLE
Update eslint config

### PR DESCRIPTION
**Pull request by edg2s**

### CODE REVIEW

**Comment by edg2s** [comment]
Failures appear unrelated to the patch.

File: package.json [hunk 1]
```diff
@@ -51,10 +51,9 @@
     "ajv": "^6.10.2",
     "bunyan": "^1.8.12",
     "coveralls": "^3.0.5",
-    "eslint": "^5.16.0",
-    "eslint-config-wikimedia": "^0.13.1",
-    "eslint-plugin-jsdoc": "^15.8.0",
-    "eslint-plugin-json": "^1.4.0",
+    "eslint": "^6.8.0",
```
Comment (Pchelolo) [inline package.json:54]: Unfortunately we can't do that yet. We still need to be able to run this on node6

**Comment by Pchelolo** [comment]
We unfortunately can't do this yet, we still need to be able to run this on node 6. eslint is not a prod dependency, but we want to be able to run tests on node 6 as well.

**Comment by edg2s** [comment]
We could disable linting tests on 6, they only need to be checked in one environment...

**Comment by Pchelolo** [comment]
Superseded by #1299

**Pull request closed without merging**